src/implementations/react/components/icon/stories/EuIndex.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
```

Comment:
Let's please keep only the button for the POC in order to be able to keep the boilerplate reduced

Reply:
Yeah, but i don't plan to merge anything from here, honestly, it's a very simple approach the one shown here and it depends on the decisions we'll make regarding all the other things, file structure, storybook configurations, build configurations..the ticket i think will still be in place for a while, surely not "fulfilling" its scope with this simple implementation.
Open to different opinions about this, anyway.